HopsanGUI/OptimizationHandler.h

```diff
@@ -77,6 +77,7 @@ class OptimizationMessageHandler : public QObject, public Ops::MessageHandler
     void objectiveChanged(size_t idx);
     void candidatesChanged();
     void candidateChanged(size_t idx);
+    void stepCompleted(size_t);
```

Comment:
The variable 'value' is missing here
Also if this method (and others) overload virtual methods in the base class, they should be marked override. 
That would lead to less risk for mistakes.

Reply:
I didn't even know there were such a keyword, but now it's there. You learn something new every day :)

Reply:
Yes it came with c++11